CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Serial terminal replaced with `menu` for simplicity
 * The broker can now be specified using DNS
 * Updated to RTIC v2 and async support. MSRV bumped to 1.75.0
+* All settings, including RF channel settings, are now exposed via the USB interface
```

Comment:
Describe the scope of the breakage and hints to adress it.

Reply:
This actually raises an interesting point that the MQTT `save` command would potentially conflict with the USB-based `save`. I'll open an issue.

Reply:
Right. Let's sort that out. Can you propose a behavioral spec of the two "save"?

Reply:
I would propose that we make the MQTT `save <ch>` command semantically equivalent to USB-based `save /booster/channel/<ch>/*`

That is, they would both trigger the same actions (currently, saving channel data to flash, not EEPROM).

Reply:
Sounds good.

Reply:
Added more description in 088b10202ef98a6dbe41c527f801c147a0cd082a. I want to hold off on fixing the `save` command till later because the change is somewhat complex.

Reply:
Tracking issue for cleaning up the MQTT `save` command is https://github.com/quartiq/booster/issues/405